deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-chrony
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,server%20129.6.15.28%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Aserver%20132.163.97.1%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Aserver%20132.163.96.1%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Adriftfile%20%2Fvar%2Flib%2Fchrony%2Fdrift%0Artcsync%0Alogdir%20%2Fvar%2Flog%2Fchrony
+          mode: 420
+          overwrite: true
+          path: /etc/chrony.conf
+  kernelArguments:
+    - loglevel=7
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-chrony
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,server%20129.6.15.28%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Aserver%20132.163.97.1%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Aserver%20132.163.96.1%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Adriftfile%20%2Fvar%2Flib%2Fchrony%2Fdrift%0Artcsync%0Alogdir%20%2Fvar%2Flog%2Fchrony
+          mode: 420
+          overwrite: true
+          path: /etc/chrony.conf
+  kernelArguments:
+    - loglevel=7
```

Comment:
what are the kernelArguments for?

Reply:
Very good point.  These should not be in there. updating config now